include/slf4ec/slf4ec.h

```diff
@@ -217,11 +219,15 @@ LogResult noLog();
     yfLog0(__FILE__, __LINE__, FUNCTION, &logCategory, level, __VA_ARGS__)
```

Comment:
Instead of all the complicated macro magic to deal with variadic macros, woudn't it be easier and cleaner to just use ##__VA_ARGS__ ?

Reply:
I wish! See comment on line 54.
Sadly this doesn't work when __VA_ARGS__ is empty. GCC does support "trailing comma swallowing" but according to C99 the trailing comma should always be kept, meaning GCC's feature is not strict C99.
And there are compiles out there known not to support this feature.

Reply:
Thanks for the clarification.